AnnoyancesFilter/sections/popups.txt

```diff
@@ -160,6 +160,7 @@ cookpad.com##.css-1c6thrl
 cookpad.com##.css-1ydy0aj
 cookpad.com##.css-37eo8b
 ipeenk.com##div[id^="notification"]
+superlegalbrinquedos.com.br##.modal--optin
```

Comment:
Actually, it will break the scrolling of the page.
You should rewrite it to the
```
superlegalbrinquedos.com.br#$#.modal--optin { display: none !important; }
superlegalbrinquedos.com.br#$#body.modal-open { overflow: auto !important; padding-right: 0 !important; }
```